akka-typed/src/main/scala/akka/typed/persistence/scaladsl/PersistentActor.scala

```diff
@@ -116,7 +125,7 @@ object PersistentActor {
 
   @InternalApi
   private[akka] case class Persist[Event, State](event: Event) extends Effect[Event, State] {
-    override val events = event :: Nil
+    override def events = event :: Nil
```

Comment:
Hm, not sure if good; does not matter much I guess though

Reply:
This comes from a previous comment from @johanandren about unnecessarily allocating a List. 

Maybe a small optimization, but it's certainly no harm. You mean it's not good because it should be a `lazy val`?

Btw, this is internal code and is never read.